packages/site-admin/src/index.ts

```diff
@@ -0,0 +1,4 @@
+/*
+ * Utils
+ */
+export * from './utils/meta';
```

Comment:
Could you elaborate on what we're planning to do with the meta utils?

Reply:
just for developing purposes for now 😬 Should I remove it?

Reply:
I wonder whether we can keep this helper here and remove it later in case we really want

Reply:
For dev, I don't mind, but usually I prefer the other way around: including things only when we need it. 

See [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).

Reply:
Let me remove it. We can restore it if we consider it useful

Reply:
removed here 5f9ed7a8f54b7992f6c0908dcff210c13a0ef518